### PR TITLE
Update `presents_super`

### PR DESCRIPTION
**Pull request by edsonk**
- Simplify option arguments
- Allow presentation of collections

### CODE REVIEW

File: lib/frosting/base_presenter.rb [hunk 1]
```diff
@@ -5,10 +5,13 @@ module Frosting
   class BasePresenter < SimpleDelegator
     include Presentation
 
-    def self.presents_super(*methods, options: {})
-      methods.each do |method|
+    def self.presents_super(*arguments)
+      options = arguments.last.is_a?(Hash) ? arguments.pop : {}
```
Comment (artfuldodger) [inline lib/frosting/base_presenter.rb]: This seems less simple and makes the method definition ambiguous.

Comment (artfuldodger) [inline lib/frosting/base_presenter.rb]: Buuut.. I like the interface it provides better. Okay, fine.

**Comment by artfuldodger** [comment]
:+1:

File: lib/frosting/base_presenter.rb (hunk 1 above)
Comment (edsonk) [inline lib/frosting/base_presenter.rb]: :kissing_cat:

**Comment by tpiekos** [comment]
:+1:

**Pull request merged**